lib/trianglify.js

```diff
@@ -82,9 +82,8 @@ function Trianglify(opts) {
         .scale(opts.x_colors || opts.y_colors)
         .mode(opts.color_space);
 
-      var gradient = opts.x_colors
-        ? function(x, y) { return scale(x) }
-        : function(x, y) { return scale(y) };
+      gradient = opts.x_colors ? function(x, y) { return scale(x); }
```

Comment:
Any reason for this whitespace change?

Reply:
When running _npm run start_, the linter complained!

Reply:
oops, probably my bad. Sorry about that!